app_test.go

```diff
@@ -62,6 +62,16 @@ func TestNewApp(t *testing.T) {
 		assert.True(t, found)
 	})
 
+	t.Run("ProvidesShutdowner", func(t *testing.T) {
+		found := false
+		app := fxtest.New(t, Invoke(func(s Shutdowner) {
+			assert.NotNil(t, s)
+			found = true
+		}))
+		defer app.RequireStart().RequireStop()
+		assert.True(t, found)
```

Comment:
Do we need the extra assertion on `found`? The rest of the unit tests are already asserting that invoked functions are always run.

Reply:
So I patterned this test off of the "ProvidesLifecycle" line for line. I don't think that the assertion is necessary, but I figured that the tests to confirm whether or not a type is provided to the container should be the same in content. lmk if you think I should remove the found assertion from both tests or leave it.